pydantic_ai_slim/pydantic_ai/toolsets/function.py

```diff
@@ -36,23 +36,27 @@ class FunctionToolset(AbstractToolset[AgentDepsT]):
     max_retries: int
     tools: dict[str, Tool[Any]]
     _id: str | None
+    _default_require_parameter_descriptions: bool
```

Comment:
I think we can drop the `_default_` prefix here, and make it a proper dataclass field.

```suggestion
    require_parameter_descriptions: bool
```